StructureDefinition/England-Organization.xml

```diff
@@ -0,0 +1,171 @@
+        <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Organization" />
+      </type>
+    </element>
+    <element id="Organization.identifier:odsSiteCode">
+      <path value="Organization.identifier" />
+      <sliceName value="odsSiteCode" />
+      <max value="0" />
+    </element>
+    <element id="Organization.type">
+      <path value="Organization.type" />
+      <definition value="The Organisation types **MUST** be present and comes from the valueSet.&#xD;&#xA;`type` will also include details on when these roles applied, the current status and a boolean to indicate the Organisations primary role. These extensions are for API's produced by NHS Digital one, the are not expected to be present in resources created by NHS providers or suppliers.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&#xD;&#xA; &quot;type&quot;:  [&#xD;&#xA;        {&#xD;&#xA;            &quot;extension&quot;:  [&#xD;&#xA;                {&#xD;&#xA;                    &quot;url&quot;: &quot;https://fhir.nhs.uk/StructureDefinition/Extension-ODS-Organization-Role&quot;,&#xD;&#xA;                    &quot;extension&quot;:  [&#xD;&#xA;                        {&#xD;&#xA;                            &quot;url&quot;: &quot;primaryRole&quot;,&#xD;&#xA;                            &quot;valueBoolean&quot;: true&#xD;&#xA;                        },&#xD;&#xA;                        {&#xD;&#xA;                            &quot;url&quot;: &quot;status&quot;,&#xD;&#xA;                            &quot;valueString&quot;: &quot;Active&quot;&#xD;&#xA;                        }&#xD;&#xA;                    ]&#xD;&#xA;                },&#xD;&#xA;                {&#xD;&#xA;                    &quot;url&quot;: &quot;https://fhir.nhs.uk/StructureDefinition/Extension-ODS-Organization-Role-ActivePeriod&quot;,&#xD;&#xA;                    &quot;extension&quot;:  [&#xD;&#xA;                        {&#xD;&#xA;                            &quot;url&quot;: &quot;activePeriod&quot;,&#xD;&#xA;                            &quot;valuePeriod&quot;: {&#xD;&#xA;                                &quot;start&quot;: &quot;2020-04-01&quot;&#xD;&#xA;                            }&#xD;&#xA;                        },&#xD;&#xA;                        {&#xD;&#xA;                            &quot;url&quot;: &quot;dateType&quot;,&#xD;&#xA;                            &quot;valueString&quot;: &quot;Operational&quot;&#xD;&#xA;                        }&#xD;&#xA;                    ]&#xD;&#xA;                }&#xD;&#xA;            ],&#xD;&#xA;            &quot;coding&quot;:  [&#xD;&#xA;                {&#xD;&#xA;                    &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/organisation-role&quot;,&#xD;&#xA;                    &quot;code&quot;: &quot;198&quot;,&#xD;&#xA;                    &quot;display&quot;: &quot;NHS TRUST SITE&quot;&#xD;&#xA;                }&#xD;&#xA;            ]&#xD;&#xA;        }&#xD;&#xA;    ],&#xD;&#xA;```" />
+      <binding>
+        <strength value="preferred" />
```

Comment:
Is the NHSE IG standard for non SNOMED to be preferred, compared to extensible in UK Core?